src/main/resources/messages/LanguageServerBundle.properties

```diff
@@ -130,6 +130,8 @@ no.usage.type.typeDefinitions=No Type Definition found.
 no.usage.type.references=No Reference found.
 no.usage.type.implementations=No Implementation found.
 
+# Type declaration provider
+goto.typeDeclaration.notFound=No type declarations found

```

Comment:
Had to add this here because there was not already a message for this in the IDE's standard message bundle as it doesn't show any message if the **Navigate | Type Declaration** action yields no results. I agree that showing such a message provides a better experience, but that's why there's a custom message here.